src/components/CouponDetails/ActionButton.jsx

```diff
@@ -5,7 +5,7 @@ import {
 } from '@edx/paragon';
 import RemindButton from '../RemindButton';
 import RevokeButton from '../RevokeButton';
-import { ACTION_TYPES, ACTION_LABELS, COUPON_FILTER_TYPES } from './constants';
+import { ACTIONS, COUPON_FILTERS, COUPON_FILTER_TYPES } from './constants';
```

Comment:
So the only usage of `COUPON_FILTER_TYPES` I see still in this file is at the very bottom in the prop definition and I'm wondering if it doesn't need to be there either?

`selectedToggle: PropTypes.oneOf(Object.values(COUPON_FILTER_TYPES)).isRequired,`

Reply:
I debated it, but I think it's valuable to have a quick and easy alert. I could define it as an array, but then you'd have to add elements to that array every time you add a type.

Reply:
Ah I didn't think of that. Good point!